src/commands/profiles/exist.ts

```diff
@@ -0,0 +1,119 @@
+import * as cosmos from "@azure/cosmos";
+import { Command, flags } from "@oclif/command";
+import cli from "cli-ux";
+import * as parse from "csv-parse";
+import * as csvStringify from "csv-stringify";
+import * as fs from "fs";
+import * as transform from "stream-transform";
+import { getCosmosConnection } from "../../utils/azure";
+
+export default class ProfilesExist extends Command {
+  public static description =
+    "Check if, for the given fiscal codes, there are relative profiles or not. It outputs the given csv with a new last column containing true if a profile exists";
+
+  public static flags = {
+    input: flags.string({
+      char: "i",
+      description: "Input file (CSV, with the CF as first column)",
+      required: true
+    }),
+    parallel: flags.integer({
+      char: "p",
+      default: 1,
+      description: "Number of parallel workers to run"
+    })
+  };
+
+  public async run(): Promise<void> {
+    const { flags: parsedFlags } = this.parse(ProfilesExist);
+
+    const inputStream = fs.createReadStream(parsedFlags.input);
+
+    const parser = parse({
+      trim: true,
+      skip_empty_lines: true
+    });
+
+    try {
+      cli.action.start("Retrieving cosmosdb credentials");
+      const { endpoint, key } = await getCosmosConnection(
+        "agid-rg-test",
+        "agid-cosmosdb-test"
+      );
+      cli.action.stop();
+
+      cli.action.start("Querying profiles...");
+      const client = new cosmos.CosmosClient({ endpoint, auth: { key } });
+      const database = client.database("agid-documentdb-test");
+      const container = database.container("profiles");
+
+      // retrieve all fiscalCodes from database
+      const response = container.items.query(
+        "SELECT c.fiscalCode FROM c WHERE c.version = 0",
+        {
+          enableCrossPartitionQuery: true
```

Comment:
this is quite expensive ($$$), since you're querying profiles by fiscal code, you can provide the partition key (that is the fiscal code), thus hitting the right partition